__tests__/users/updateProfilePicture.mjs

```diff
@@ -0,0 +1,25 @@
+import { describe, it, expect } from "@jest/globals";
+import * as client from "../../src/clients/clients.mjs";
+import { objectStatusCodes } from "../../src/lib/updateProfilePicture/updateProfilePictureObjects.mjs";
+
+describe("Update profile picture", () => {
+    objectStatusCodes.forEach((obj) => {
+        it(`Should ${obj.it}`, async () => {
+          const result = await client.updateProfilePicture(obj.userId, obj.newProfilePicture);
+    
+          expect(result.statusCode).toEqual(obj.expect);
+        });
+      });
+    
+      it(`Should return new profile picture`, async () => {
+        const userId = "jest_user_1";
+        const newProfilePicture = "https://picsum.photos/400";
```

Comment:
laat het nummer altijd random zijn of kijkt het niet na of het de zelfde foto is? (in de BE bedoel ik)

Reply:
De seeding van de users op het begin van de testen => 
**userId**: jest_user_1
**profilePicture**: https://example.com/profile/johndoe.jpg

De test moet dan zogezegd de profile picture veranderen in: https://picsum.photos/400

Dus de nieuwe hoeft toch niet telkens anders te zijn? Het zal toch altijd verschillen van de seeding data.

Reply:
Ow ja juist, dat klopt